README.md

```diff
@@ -33,7 +33,7 @@ Tensor search uses deep-learning to transform documents, images and other data i
 ## Getting started
 
 1. Marqo requires docker. To install docker go to https://docs.docker.com/get-docker/
-2. Use docker to run Marqo:
+2. Use docker to run Marqo (M1 users will need to [go here](#M1-Mac-users)):
```

Comment:
It may not be clear what M1 means:  perhaps "Mac M1 chip users  ...

Reply:
done